subcell_pipeline/analysis/compression_metrics/constants.py

```diff
@@ -0,0 +1,5 @@
+ABSOLUTE_TOLERANCE: float = 1e-6
```

Comment:
```suggestion
ABSOLUTE_TOLERANCE: float = 1e-6
"""Tolerance level for ???"""
```

Add docstrings after each of these constants so they show up in the documentation.

Reply:
Fixed in [f8ffc9](https://github.com/simularium/subcell-pipeline/pull/58/commits/f8ffc9bc7f8a2e06701ffbdfdb6849aa836ee4b2)